### DISCUSSION TITLE
Skin editor shows internal names for components (ie. "Default" prefix where elements are actually "Triangles")

### DISCUSSION THREAD

**Discussion by shiumano** [ppy/osu#30007] [Ideas]
None of the elements with "default" in their name will be used by default.
The current default skin is argon, and none of the elements with "default" in there are used by default.
Not long ago, only the key counter was used in classic and could be called "default," but it is no longer used.
The default rank display is not used in any of the default skins.

In addition, only the pp counter has "triangles" on it, which is not consistent.

![Screenshot_20240926-223227~2](https://github.com/user-attachments/assets/31a568ec-a185-4b17-a050-195673d5551e)

**Comment by peppy** [comment]
We'll probably add friendly naming for these in the future. "Default" internally now usually means "Triangles".
